packages/base-map/src/index.tsx

```diff
@@ -91,8 +91,14 @@ const BaseMap = ({
 
   const toggleableLayers = Array.isArray(children)
     ? children
-        .flat()
-        .filter(child => child?.props?.id !== undefined)
+        .flat(10)
+        .filter(
+          child =>
+            child?.props?.id !== undefined &&
+            // Some sources will not have layers as children, and should be ignored
+            // from the list.
+            child?.props?.alwaysShow !== true
```

Comment:
Rename `alwaysShow` to `notListed`? Or, if this condition is targeting `<Source>` elements, could we check `child?.type.name` and `child?.props?.children` directly?

<img width="300" alt="Screenshot 2022-12-21 at 9 24 04 AM" src="https://user-images.githubusercontent.com/56846598/208927762-6304bf74-d6a3-40df-a25d-925b3cf403cb.png">

Reply:
It's more common to want to have a layer off by default than on so I did it this way to avoid writing out too many props. 

It's also used on `Layer`s that shouldn't be ridable!